ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -38,6 +38,25 @@ def softsign(x):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
 
 
+@with_supported_dtypes(
+    {
+        "2.0 and below": (
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "float16",
+            "float32",
+            "float64",
+        )
```

Comment:
Sorry I haven't noticed this before in the earlier review, but where did you find this info? It sounds correct but I can't seem to find it anywhere in their docs or code...

Reply:
Got it, thank you!

Reply:
Please let me know of any further requirements 😄